src/react/native/api/ToastAndroid.hx

```diff
@@ -0,0 +1,21 @@
+package react.native.api;
+
+@:jsRequire('react-native', 'ToastAndroid')
+extern class ToastAndroid {
+	static function show(message:String, duration:Duration):Void;
+	static function showWithGravity(message:String, duration:Gravity, gravity:Float):Void;
+}
+
+@:enum
+abstract Duration(Int) {
+    var SHORT = 0;
+    var LONG = 1;
+}
+
+@:enum
+abstract Gravity(Int) {
+    var TOP = 0;
```

Comment:
Not sure about these values:
https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/modules/toast/ToastModule.java#L50-L54